.NET/Microsoft.Recognizers.Text.DataTypes.TimexExpression/TimexHelpers.cs

```diff
@@ -254,6 +257,46 @@ public static TimeRange TimeRangeFromTimex(TimexProperty timex)
             };
         }
 
+        public static string GenerateCompoundDurationTimex(List<string> timexList)
+        {
+            var isTimeDurationAlreadyExist = false;
+            var timexBuilder = new StringBuilder(Constants.GeneralPeriodPrefix);
+
+            foreach (string timexComponent in timexList)
+            {
+                // The Time Duration component occurs first time
+                if (!isTimeDurationAlreadyExist && IsTimeDurationTimex(timexComponent))
+                {
+                    timexBuilder.Append($"{Constants.TimeTimexPrefix}{GetDurationTimexWithoutPrefix(timexComponent)}");
+                    isTimeDurationAlreadyExist = true;
+                }
+                else
+                {
+                    timexBuilder.Append($"{GetDurationTimexWithoutPrefix(timexComponent)}");
+                }
+            }
+
+            return timexBuilder.ToString();
```

Comment:
Invariant culture here.

Reply:
stringBuilder.Tostring has no IFormatProvider parameter, can I use AppendFormat to assure Ivariant Culture instead?

Reply:
Interesting. Leave this one as-is then.